bulk-update/bulk-update.js

```diff
@@ -97,7 +113,16 @@ export async function loadListData(source, fetchFunction = fetch, fetchWaitMs =
  */
 export default async function main(config, migrate, reporter = null) {
   config.reporter = reporter || config.reporter;
-  const { length } = config.list;
+  const { list, outputDir, siteUrl, stagePath, locales } = config;
+  const { length } = list;
+
+  if (outputDir) {
+    fs.mkdirSync(outputDir, { recursive: true });
+    fs.writeFileSync(`${outputDir}/list.json`, JSON.stringify(config.list, null, 2));
+
+    const output = list.map((entry) => [`${siteUrl}${entry}`, localizedStageUrl(siteUrl, entry, stagePath, locales)]);
+    fs.writeFileSync(`${outputDir}/staged.json`, JSON.stringify(output, null, 2));
```

Comment:
Should this be optional, only if stagePath exists?
```suggestion
    if (stagePath) {
      const output = list.map((entry) => [`${siteUrl}${entry}`, localizedStageUrl(siteUrl, entry, stagePath, locales)]);
      fs.writeFileSync(`${outputDir}/staged.json`, JSON.stringify(output, null, 2));
    }
```

Reply:
Thanks!